homeassistant/components/plugwise/sensor.py

```diff
@@ -444,4 +506,4 @@ def __init__(
     @property
     def native_value(self) -> int | float | None:
         """Return the value reported by the sensor."""
-        return self.device["sensors"].get(self.entity_description.key)
+        return self.entity_description.value_fn(self.device["sensors"])
```

Comment:
How come it's OK to replace `dict.get` with the value functions which will raise `KeyError` if the key is missing?

Reply:
The get-function is not needed here because the presence of the sensors-dict has been checked in `async_setup_entry()`.